steps/trigger_job/trigger_job.py.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "type": "object",
+    "properties": {
+        "instance_url": {
+            "description": "Base URL for the inteligence module",
+            "type": "string",
+            "enum": [
```

Comment:
Melhor deixar sem enum, porque ai a pessoa que for utilizar o step, não precisa mexer no schema para conseguir utilizar...  Pra todos os fields porque ai fica bem genérico.